### ISSUE TITLE
Question about QM9 usage

### ISSUE THREAD

**Issue by loilisxka** [atomistic-machine-learning/schnetpack-gschnet#16]
I'm executing
`gschnet_train --config-dir=/home/huzheyu/ai_lab/gener_ai/test experiment=gschnet_qm9`
but get the error
```
Cannot find primary config 'train'. Check that it's in your config search path.

Config search path:
      provider=hydra, path=pkg://hydra.conf
      provider=main, path=pkg://schnetpack.configs
      provider=hydra-colorlog, path=pkg://hydra_plugins.hydra_colorlog.conf
      provider=command-line, path=file:///home/huzheyu/ai_lab/gener_ai/test
      provider=schema, path=structured://

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
```

I don't know what causes this error, if you guys can answer I'd be very grateful.

**Comment by NiklasGebauer** [comment]
Hello @loilisxka ,

thanks for reporting this issue! We rely on the training procedure give in schnetpack and it seems like something is not working as intended. I was able to reproduce the bug by installing schnetpack-gschnet with the newest version of schnetpack (2.1.0). 
This version of schnetpack was released last week and I suppose some changes have led to this error. I couldn't pinpoint the exact cause yet, but I will try to find a proper solution as soon as possible.

As a workaround, you can uninstall schnetpack
```pip uninstall schnetpack```
and install the previous version in your environment
```pip install schnetpack==2.0.4```
This should enable you to properly start the training.

Hope this helps and sorry for the hiccups with the new version!
Best regards,
Niklas

**Comment by NiklasGebauer** [comment]
The bug was caused by the setup of schnetpack, where config files were not properly copied into the package. We fixed this in the schnetpack repository and released [version 2.1.1](https://github.com/atomistic-machine-learning/schnetpack/releases/tag/v2.1.1). 

So it should be possible to upgrade to schnetpack 2.1.1 now and run the training as expected.
Please let me know in case this fix does not work in your case.

Best regards,
Niklas

**Comment by loilisxka** [comment]
OK, It's working, thanks for you very much.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "atomistic-machine-learning/schnetpack-gschnet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月5日(星期四) 晚上7:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [atomistic-machine-learning/schnetpack-gschnet] Question about QM9 usage (Issue #16)





 
The bug was caused by the setup of schnetpack, where config files were not properly copied into the package. We fixed this in the schnetpack repository and released version 2.1.1.
 
So it should be possible to upgrade to schnetpack 2.1.1 now and run the training as expected.
 Please let me know in case this fix does resolve your problem.
 
Best regards,
 Niklas
